content/javascript/concepts/operators/operators.md

```diff
@@ -163,6 +163,71 @@ variable = condition ? assignedIfTrue : assignedIfFalse;
 
 This operator combines the functionalities of the assignment, comparison, and logical operators.
 
+The ternary operator can also perform multiple steps based on a certain condition, similar to an `if...else` condition statement.
+
+The key point to focus on is the syntax. Pay special attention to where the `()`s are placed. A developer may make the mistake of using too many `()`s or placing the `()`s incorrectly.
+
+### Multi-Step Ternary Operator Examples
+
+For example, the following `if...else` statement...
+
+```js
+let i = true;
+let a = 0;
+let b = 0;
+
+if (i == true) {
+    a++;
+    b++;
+} else {
+    a = 0;
+    b = 0;
+}
+```
+
+...can be translated into the following two-step ternary operator!
+
+```js
+let i = true;
+let a = 0;
+let b = 0;
+
+i == true ? (a++, b++) : (a = 0, b = 0);
+console.log('a = ', a, 'b = ', b);
+```
+
+The ternary operator can also perform as many steps as you need.
+
+```js
+let i = true;
+let a = 0;
+let b = 0;
+let c = 0;
+// ...
+
+i == true ? (a++, b++, c++) : (a = 0, b = 0, c = 0);
+console.log('a = ', a, 'b = ', b, 'c = ', c);
+```
```

Comment:
```suggestion
This will output the following:
```

And then we should add a \```shell output snippet that looks like this:

```
a = 1 b = 1
a = 2 b = 2
a = 3 b = 3 c = 1
```